tests/conftest.py

```diff
@@ -79,6 +82,10 @@ def pytest_runtest_call(item):
 
 def pytest_collection_modifyitems(session, config, items):
     from firedrake.utils import SLATE_SUPPORTS_COMPLEX
+    from firedrake.pytorch_coupling import get_backend
+
+    backend = get_backend("pytorch")
```

Comment:
Could you maybe specify `ml_backend` (or similar) instead of just "backend". We're reusing the same term for CPU/OpenCL/CUDA.